src/controllers/accounts.js

```diff
@@ -1,26 +1,15 @@
 /* @flow */
 'use strict'
 
-const _ = require('lodash')
-const db = require('../services/db')
-const log = require('../services/log')('accounts')
-const config = require('../services/config')
 const request = require('five-bells-shared/utils/request')
-const NotFoundError = require('five-bells-shared/errors/not-found-error')
-const UnauthorizedError =
-require('five-bells-shared/errors/unauthorized-error')
-const Account = require('../models/account').Account
+const model = require('../models/accounts')
 
-exports.getCollection = function * find () {
-  const accounts = yield Account.findAll()
-  this.body = _.invoke(accounts, 'getDataExternal')
+function * getCollection () {
+  this.body = yield model.getAccounts()
```

Comment:
I don't see the `/connectors` endpoint in the doc. Is this documented somewhere?